specification/src/SUMMARY.md

```diff
@@ -3,6 +3,10 @@
 [Welcome](./welcome.md)
 [Overview](./overview.md)
 
+# Development
+- [Introduction](./setup/introduction.md)
+- [Compiling binaries](./setup/compilation.md)
```

Comment:
This should be `development` instead of `setup`, or the source files should be moved. Both are fine by me.